Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM registry.opensuse.org/opensuse/leap:15.3
+
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 UWSGI_WSGI_FILE=/pcw/webui/wsgi.py UWSGI_MASTER=1
+ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSGI_ENV_BEHAVIOR=holy
+
+## System preparation steps ################################################# ##
+
+COPY . /pcw/
+
+# We do the whole installation and configuration in one layer:
+# * Install system requirements
+RUN zypper -n in python3 python3-devel python3-pip gcc && \
+    zypper -n ar http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.3/ SUSE_CA && \
+    zypper -n in ca-certificates-suse && rm -rf /var/cache
```

Comment:
as you already have `.dockerignore` no need to do any manual cleanups ?

Reply:
also if you will first add repo and than do install call you may put all packages in one zypper call

Reply:
I merged the `zypper` calls together, however the `rm -rf /var/cache` is still required, because that's outside the `/pcw` directory and those contains the system (and zypper) cache and thus nothing we can control via `.dockerignore`.